open-source/overview.mdx

```diff
@@ -2,22 +2,22 @@
 
 title: Open Source Overview
 sidebarTitle: Overview
-description: Installation and configuration of Edgee proxy
-
+description: Edgee Open Source Edition
+icon: github
 ---
 
-Make sure you've read [introduction](https://docs.edgee.cloud/introduction) if you want to understand the main concepts and the architecture of Edgee.
+The full documentation and the proxy implementation is [available on GitHub](https://github.com/edgee-cloud/edgee).
 
-# Running Edgee
+# How to install Edgee
 
-The next section will explain how to configure Edgee proxy, for now let's assume you already have a valid configuration
-file. For all examples we're gonna assume that TLS certificates can be found in `/var/edgee/cert` and WebAssembly
-components in `/var/edgee/wasm`.
+Once you have a valid configuration file (see next section), you can run Edgee in different ways, 
+using the installer, Docker or running as a Rust crate.
 
 <CodeGroup>
 ```shell Installer script
-curl https://install.edgee.cloud | sh
-# Edgee executable is now at ./edgee
+curl --proto '=https' --tlsv1.2 -sSf https://install.edgee.cloud | sh
```

Comment:
Why not keep the simplest command line `curl https://install.edgee.cloud | sh` ?

Reply:
I have aligned this to what we have on https://github.com/edgee-cloud/edgee?tab=readme-ov-file#using-the-installer

Reply:
Yes, but I think we also need to change it in the readme. The same is true for the Cargo install.